client/src/js/agreement/AgreementDisplay.jsx

```diff
@@ -1,104 +1,45 @@
-        litigation against You or any other entity (including a cross-claim or counterclaim in a lawsuit) alleging that
-        Your Contribution, or the Work to which You have contributed, constitutes direct or contributory patent
-        infringement, then any patent licenses granted to that entity under this Agreement for that Contribution or Work
-        shall terminate as of the date such litigation is filed.
-      </p>
-      <p>4. <b>Right to Grant Licenses. </b>You represent that You are legally entitled to grant the above
-        licenses. &nbsp;If your employer(s) has (have) rights to intellectual property that you create that includes
-        your Contributions, you represent that you have received permission to make Contributions on behalf of that
-        employer, that your employer has waived such rights for your Contributions to ON.Lab and ONF, or that your
-        employer has executed a separate Corporate CLA with ONF.
-      </p>
-      <p>5. <b>Original Creation.&nbsp; </b>You represent that each of Your Contributions is Your original creation (see
-        section 7 for submissions on behalf of others).&nbsp; You represent that Your Contribution submissions include
-        complete details of any third-party license or other restriction (including, but not limited to, related patents
-        and trademarks) of which you are personally aware and which are associated with any part of Your Contributions.
-      </p>
-      <p>6. <b>Support; Warranty Disclaimer.&nbsp; </b>You are not expected to provide support for Your Contributions,
-        except to the extent You desire to provide support. You may provide support for free, for a fee, or not at all.
-        Unless required by applicable law or agreed to in writing, You provide Your Contributions on an "AS IS" BASIS,
-        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any
-        warranties or conditions of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.
-      </p>
-      <p>7. <b>Contributions That Are Not Your Original Creation.&nbsp; </b>Should You wish to submit work that is not
-        Your original creation, You may submit it to ONF separately from any Contribution, identifying the complete
-        details of its source and of any license or other restriction (including, but not limited to, related patents,
-        trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as
-        "Submitted on behalf of a third-party: [named here]".
-      </p>
-      <p>8. <b>Required Notifications. </b>You agree to notify ONF of any facts or circumstances of which you become
-        aware that would make these representations inaccurate in any respect.
-      </p>
+      {
+        props.text ?
+          <div className={classes.claContainer} dangerouslySetInnerHTML={{ __html: props.text }}/>
```

Comment:
If storing HTML is so dangerous, would it make sense to store markdown and use a renderer such as this one: https://github.com/rexxars/react-markdown

Reply:
Nice find.. I'll add `rect-markdown`